TCC.Lib/TarCompressCrypt.cs

```diff
@@ -174,10 +168,10 @@ private static void PrepareDecryptionKey(Block block, TccOption option, out stri
                 // crypt passfile
                 string publicKeyPath = Path.Combine(Environment.CurrentDirectory, option.PublicPrivateKeyFile);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused 'publicKeyPath' local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1481)